guides/idea_setup.md

```diff
@@ -0,0 +1,36 @@
+# IntelliJ IDEA Project Setup Guide
+
+Follow these detailed steps to set up your project in IntelliJ IDEA seamlessly:
+
+1. Import Project from GitHub:
+    - Begin by importing the project from GitHub into IntelliJ.
+
+2. Build with Maven. Run the following commands from the command line:
+    - `mvn clean install -f dhis-2/pom.xml`
+    - `mvn clean install -f dhis-2/dhis-web/pom.xml`
+

```

Comment:
This section is the n.1. We should always install the project first at the command line. This will save time because otherwise, IntelliJ will do it for us by downloading all the libraries, and with a UI, it can take a lot of time, especially if it is the first import of a project.
We can add the `-DskipTests` so it will skip tests and it will install much faster